sending email note (#1819)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Modules/Automation_module/Automation_script_actions/Email.md

   > [!NOTE]
   >
   > - If you want to specify multiple indices for one table parameter, use a semicolon “;” as separator.

>    > - If you want to specify multiple parameters for one element, service, or protocol version, assign them all within a single line.

Author: @tmsrosa, should ";" also be used as a separator here, similar to when specifying multiple indices for a table parameter? (see suggested change below)

Reviewer: It should not.
Purpose of the additional info is to avoid that users create multiple lines for the same element/service/protocol.

Here's an example of how it should:
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/31303691/1a832a7f-03e3-4fc9-a597-8473fb4ab963)

>    > - If you want to specify multiple parameters for one element, service, or protocol version, assign them all within a single line.

Author: ```suggestion
   > - If you want to specify multiple parameters for one element, service, or protocol version, use a semicolon ";" as separator.
```

Reviewer: It should not mention the ;. it should mention the original text.
